package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@salesforce/lwc-dev-mobile-core",
     "description": "Core module supporting Salesforce CLI mobile extension plug-ins",
-    "version": "4.0.0-alpha.3",
+    "version": "4.0.0-alpha.4",
```

Comment:
Bumping the alpha's version number so it can be manually published for immediate consumption.